app/controllers/bills_controller.rb

```diff
@@ -28,9 +28,9 @@ def create
 
   def update
     if @bill.update status: bill_params[:status]
-      flash[:success] = t(".update_s")
+      flash[:success] = @bill.cancle? ? t(".update_s") : t(".payment_s")
```

Comment:
- cancle nghĩ là  gì?
- sao cancel rồi còn update được?

Reply:
bill của em có 3 trạng thái; e đặt trong biết enum: {cancle: 2, active: 1, unactive: 0}
- thanh toán thì status của bill là unactive: 0
- Hủy đơn update status của bill là cancle: 2
Với mỗi hiểu thì em sẽ thông báo khác nhau. nên em dùng toàn tử ? để check.

Reply:
- cancle sai  chính tả nhé
- Tại sao người dùng lại được quyền cancel? Chỉ có admin mới có quyền update  hủy đơn chứ